src/component.js

```diff
@@ -190,7 +191,10 @@ let prevDebounce;
  */
 export function enqueueRender(c) {
 	if (
-		(!c._dirty && (c._dirty = true) && rerenderQueue.push(c) === 1) ||
+		(!c._dirty &&
+			(c._dirty = true) &&
+			rerenderQueue.push(c) &&
+			!rerenderCount++) ||
```

Comment:
We might be lacking a test case for this change. All tests keep passing when I remove the `rerenderCount` check.

Reply:
Good point. I added a test to browser/render.test.js that passes with this PR, but fails on master or if you remove the `rerenderCount` check.